app/models/placement_suitability.rb

```diff
@@ -1,9 +1,12 @@
 class PlacementSuitability < ApplicationRecord
+  include Geocodeable
+
   belongs_to :foster_parent, inverse_of: :placement_suitability
 
   validates_with AnyBooleanValidator, fields: PlacementNeed::OPTIONS
 
   before_validation :sanitize_input, :sanitize_postcode
+  after_validation :geocode
```

Comment:
This is not needed as it's part of the `Geocodeable` concern.

Reply:
removed

Reply:
Yep whenever we create placement_need or suitability it gets the lat/long attributes set via the geocode call. We can also set it to geocode where is an address change, although that may not be able to be included in the concern due to the different address attributes  of needs and suitablities...